feature/notification/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="notification_channel_name_default">Not categorized</string>
+    <string name="notification_channel_name_start_favorite_session">start favorite session</string>
+    <string name="notification_channel_name_announcement">Announcement</string>
```

Comment:
🙏

Reply:
OK. Should I restore the deleted Topic class?

Reply:
You do not need to revert. 🙏 
I just want to remove announcement-related changes from this PR. 🙇

Reply:
Deletion complete